dot/peerset/test_helpers.go

```diff
@@ -44,7 +44,7 @@ func newTestPeerSet(t *testing.T, in, out uint32, bootNodes, reservedPeers []pee
 
 	handler.AddPeer(0, bootNodes...)
 	handler.AddReservedPeer(0, reservedPeers...)
-	time.Sleep(time.Millisecond * 4000)
+	time.Sleep(time.Millisecond * 100)
```

Comment:
are we sure lowering these sleep duration won't cause instabilities when testing with slow machines (:eyes: our CI)?

Reply:
TBH, I think I have done that by mistake.
But now that I have added `discover` function in `AddReservedPeer`, we would not need a long delay.